fiboa_cli/convert_utils.py

```diff
@@ -482,13 +492,14 @@ def convert(self, output_file, cache=None, input_files=None, source_coop_url=Non
         if not original_geometries:
             gdf.geometry = gdf.geometry.make_valid()
             gdf = gdf.explode()
+            gdf = gdf[np.logical_and(gdf.geometry.type == "Polygon", gdf.geometry.is_valid)]
             if gdf.geometry.array.has_z.any():
                 log("Removing Z geometry dimension", "info")
                 gdf.geometry = gdf.geometry.force_2d()
 
         gdf = self.post_migrate(gdf)
 
-        if hash_before != hash_df(gdf):
+        if hash_before != hash_df(gdf.head()):
```

Comment:
How fast is this hashing? We could potentially also just always print the head here...

Reply:
Hashing is fast because we only use head. Now we only print again if something has changed (makes debugging easier).